sycl/include/sycl/handler.hpp

```diff
@@ -1526,6 +1526,12 @@ class __SYCL_EXPORT handler {
     setType(detail::CG::CodeplayHostTask);
   }
 
+  /// @brief Get the command graph if any associated with this handler. It can
+  /// come from either the associated queue or from being set explicitly through
+  /// the appropriate constructor.
+  std::shared_ptr<ext::oneapi::experimental::detail::graph_impl>
+  getCommandGraph() const;
+
```

Comment:
Thankfully this isn't a breaking change, but it is a ABI change. There's a new symbol that needs added to `abi/sycl_symbols_linux.dump`

Reply:
I actually just pushed a fix to this branch quickly, since I have the command-line for generating symbols in my bash history